Gems/WhiteBox/Code/Source/Viewport/WhiteBoxEdgeTranslationModifier.cpp

```diff
@@ -65,18 +66,17 @@ namespace WhiteBox
     // (ensure to remove duplicates as vertices will be shared across edges)
     static Api::VertexHandles VertexHandlesForEdges(const WhiteBoxMesh& whiteBox, const Api::EdgeHandles& edgeHandles)
     {
-        auto vertexHandles = std::reduce(
-            edgeHandles.cbegin(), edgeHandles.cend(), Api::VertexHandles{},
-            [&whiteBox](Api::VertexHandles vertexHandles, const Api::EdgeHandle edgeHandle)
+        Api::VertexHandles vertexHandles = AZStd::accumulate(
+            edgeHandles.begin(), edgeHandles.end(), Api::VertexHandles{},
+            [&whiteBox](Api::VertexHandles vertexHandles, const Api::EdgeHandle edgeHandle) 
```

Comment:
nit: It looks like there's trailing whitespace here (please could we remove) and I think we can leave the `cbegin` and `cend` calls